JLCoordinator/Sources/Presenter/PresenterImplementations/ModalPresenter.swift

```diff
@@ -5,18 +5,25 @@ open class ModalPresenter: ModalPresenting {
 
     public let presentingViewController: UIViewController
     private let adaptivePresentationDelegateWrapper: AdaptivePresentationControllerDelegateWrapper = .init()
+    private let modalPresentationConfiguration: ModalPresentationConfiguration
 
-    public init(presentingViewController: UIViewController) {
+    public init(
+        presentingViewController: UIViewController,
+        modalPresentationConfiguration: ModalPresentationConfiguration = .default
```

Comment:
(Personally) I think `modalPresentationConfiguration:` is too long. Can we keep this short?

Reply:
I renamed it to `configuration` in the initialiser of both modal presenter.